framework/src/types.ts

```diff
@@ -143,3 +144,29 @@ export interface ActionInfoForBus {
 	readonly module: string;
 	readonly name: string;
 }
+
+/* Start codec */
+export interface CodecSchema {
```

Comment:
why this is moved to types here? If used only once, prefer to keep it where it's used